### DISCUSSION TITLE
Alt + Shift + some_key in macOS not working as expected

### DISCUSSION THREAD

**Discussion by y3owk1n** [ghostty-org/ghostty#3622] [Q&A]
I am mapping alt+shift+m to maximize (or Fill in this case) my window via macOS built-in window manager as per the screenshot below.
![image](https://github.com/user-attachments/assets/5639c19b-1c1f-48d3-bb11-f67c25c02ef5)

However, when the window is focused on ghostty terminal, this key bind won't work, despite i have unbinded all of the keys.
```
// .config/ghostty/config
macos-option-as-alt = true

keybind = clear
keybind = alt+shift+m=unbind
```

Not only for this key tho, I have a couple more mappings that rely on alt+shift+some_key does not work only in ghostty terminal. Wonder what is the cause here. Help is appreciated~

How to reproduce:
1. Go to keyboard > All applications > bind any action to alt+shift+any_key
2. Go to any app that is opened other than ghostty and try the command (it should work)
3. Go to ghostty and try the same command (it should not work)

Edit:
It works for both below:
hyper+m
ctrl+shift+m 

Another edit:
This is not the same as other discussions where controls does not pass to another program like zellij. The keybinding that I mentioned are actually straight from macOS itself.

**Comment by TeriyakiBomb** [comment]
I'm having the same issue with kakoune

**Reply by jasha-hrp** [reply]
I'm having the same issue.
